modules/cli-options/src/main/scala/scala/cli/commands/FmtOptions.scala

```diff
@@ -42,7 +46,9 @@ final case class FmtOptions(
     scalafmtArg: List[String] = Nil,
 
   @Group("Format")
-    dialect: Option[String] = None
+    scalafmtDialect: Option[String] = None,
```

Comment:
since you're changing the option name here, might be worth adding an alias for the old name.
I'd add a help message, too, while I was at it.
```suggestion
  @HelpMessage("Pass a global dialect for scalafmt. This overrides whatever value is configured in the .scalafmt.conf file.")
  @Name("dialect")
    scalafmtDialect: Option[String] = None,
```